mifosng-android/src/main/res/layout/fragment_saved_collection_sheet.xml

```diff
@@ -8,7 +8,7 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:text="@string/no_saved_sheet"
-        android:textSize="40dp"
+        android:textSize="22sp"
```

Comment:
Please don't use hardcoded values. I think we have this value in the `dimens.xml` file. If it is not there, first, add then use it.

Reply:
Thanks